### PR TITLE
tap profile picture and username to navigate to profile on mobile

### PR DESCRIPTION
**Pull request by lazynina**
also add verification badges to message thread view header's on mobile.

### CODE REVIEW

**Comment by ourielohayon** [comment]
nice one. i agree this should be solved

File: src/app/messages-page/messages-page.component.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
 <!--
   MOBILE
   -->
-<div class="global__mobile flex-column">
+<div class="global__mobile flex-column" *ngIf="globalVars.isMobile()">
```
Comment (maebeam) [inline src/app/messages-page/messages-page.component.html:31]: Thx for this awesome perf fix!

File: src/app/messages-page/messages-page.component.html [hunk 2]
```diff
@@ -61,8 +61,17 @@
...
+        [routerLink]="selectedThread.ProfileEntryResponse ? AppRoutingModule.profilePath(selectedThreadDisplayName) : []"
+        queryParamsHandling="merge"
       ></div>
-      {{ selectedThreadDisplayName }}
+      <a class="link--unstyled"
+        [routerLink]="selectedThread.ProfileEntryResponse ? AppRoutingModule.profilePath(selectedThreadDisplayName) : []"
+        queryParamsHandling="merge"
+      >{{ selectedThreadDisplayName }}</a>
+      <span *ngIf="selectedThread.ProfileEntryResponse?.IsVerified" class="ml-1 text-primary">
+            <i class="fas fa-check-circle fa-md align-middle"></i>
```
Comment (maebeam) [inline src/app/messages-page/messages-page.component.html:73]: I think you might need to enable eslint

**Review by maebeam — APPROVED** [review]

**Comment by tijno** [comment]
awesome been waiting for this change to go live - makes it a lot easier to block PM spam accounts.

**Pull request merged**